api-specs/api_spec_PGS.yaml

```diff
@@ -958,4 +1010,110 @@ components:
           type: string
           example: "RequestId 77e1c83b-7bb0-437b-bc50-a7a58e5660ac has not been found"
       required:
-        - correlationId
\ No newline at end of file
+        - correlationId
+    CreditCardAuthRequest:
+      required:
+        - idTransaction
+        - reqRefNumber
+        - amount
+        - pan
+        - securitycode
+        - expireDate
+        - holder
+        - circuit
+        - threeDsData
+        - emailCh
+        - isFirstPayment
+        - idPsp
+      type: object
+      properties:
+        idTransaction:
+          type: string
+          description: transaction id on Payment Manager
+          example: "123456"
+        reqRefNumber:
+          type: string
+          example: "reqRefNumber"
+        amount:
+          type: number
+          example: 12456
+        pan:
+          type: string
+          description: Pan of the card
+          example: 16589654852
+        securitycode:
+          type: string
+          description: Card verification code
+          example: 123
+        expireDate:
+          type: string
+          description: Expiration date of the card, yyyyMM format
+          example: 203012
+        holder:
+          type: string
+          example: "Mario rossi"
+        circuit:
+          type: string
+          example: "AMEX"
+        threeDsData:
+          type: string
+          example: "threeDsData"
+        emailCh:
+          type: string
+          example: "mariorossi@gmail.com"
+        isFirstPayment:
+          type: boolean
+          example: true
+        idPsp:
+          type: string
+          example: "23456789876"
+    CreditCardAuthResponse:
+      type: object
+      required:
+        - requestId
+        - urlRedirect
+        - status
+        - timeStamp
+      properties:
+        requestId:
+          type: string
+          description: payment request Id
+        urlRedirect:
+          type: string
+          description: redirect URL generated by PGS logic
+          example: "urlRedirect.com"
+        status:
+          type: string
+          description: status
+          example: CREATED
+        timeStamp:
+          type: string
+          description: timeStamp
+          example: 1667484646944
+    CreditCardAuthResponse400:
```

Comment:
Not a blocker, maybe a more descriptive name? (e.g. `CreditCardAuthResponseBadRequest`)